rest/soundboard_sounds.go

```diff
@@ -13,7 +13,7 @@ func NewSoundboardSounds(client Client) SoundboardSounds {
 
 type SoundboardSounds interface {
 	GetSoundboardDefaultSounds(opts ...RequestOpt) ([]discord.SoundboardSound, error)
-	GetGuildSoundboardSounds(guildID snowflake.ID, opts ...RequestOpt) ([]discord.SoundboardSound, error)
+	GetGuildSoundboardSounds(guildID snowflake.ID, opts ...RequestOpt) (ItemsResponse[discord.SoundboardSound], error)
```

Comment:
why break this?

Reply:
doesn't break anything and we need the unmarshaller

Reply:
you changed the return type, this is breaking

Reply:
it's still a slice

Reply:
just unexport the type and make it an internal matter, no need to expose this outside